### PR TITLE
🐛 Fix adding duplicate activities

### LINKED ISSUE #272: Can insert duplicate activities
There's no check on whether the activity we're trying to add already exists for that community business.

Related: the query run for the data on the `/admin/users` page doesn't filter out "deleted" activities, is this deliberate @RogeredBacon?

### PR DESCRIPTION
**Pull request by astroash**
- Add query to check if it exists
- with tests
- Display error message on frontend
- Forward if any requests are made when admin token is expired
- Add error styled component to text base
Fixes #272

### CODE REVIEW

File: react-backend/client/src/cb-admin/pages/Activities.js [hunk 1]
```diff
@@ -170,23 +181,29 @@ export default class ActivitiesPage extends React.Component {
...
 
   render() {
     const { errors } = this.state;
-
+    const errorMessage = <ActivitiesError> {errors.general} </ActivitiesError>;
     return (
       <FlexContainerCol expand>
         <Nav>
-          <HyperLink to="/admin"> Back to dashboard </HyperLink>
+          <FlexItem>
```
Comment (eliascodes) [inline react-backend/client/src/cb-admin/pages/Activities.js]: Why does this need to be wrapped?

File: react-backend/routes/activities_add.js [hunk 2]
```diff
@@ -1,15 +1,30 @@
...
 
-router.post('/', (req, res, next) => {
+router.post('/', validate(schema),async (req, res, next) => {
   const activityToAdd = req.body.name;
   const pgClient = req.app.get('client:psql');
+  try {
+    const exists = await activityCheckExists(pgClient, activityToAdd, req.auth.cb_id);
+    
+    if (exists) {
+      return res.status(409).send({ error: 'Activity already exists' });
```
Comment (eliascodes) [inline react-backend/routes/activities_add.js]: Use `Boom` instead

File: react-backend/routes/activities_add.js [hunk 3]
```diff
@@ -1,15 +1,30 @@
...
 
-const router = express.Router();
+const schema = {
+  body: {
+    name: Joi.string().required(),
+  },
+};
 
-router.post('/', (req, res, next) => {
+router.post('/', validate(schema),async (req, res, next) => {
```
Comment (eliascodes) [inline react-backend/routes/activities_add.js]: Space?

File: react-backend/client/src/cb-admin/pages/Activities.js (hunk 1 above)
Comment (astroash) [reply react-backend/client/src/cb-admin/pages/Activities.js]: Otherwise the title wasn't centered.

**Pull request merged**